From 7b31761c9ab439fd723d5b844e39cbe7a531dac4 Mon Sep 17 00:00:00 2001
From: Rehan Durrani <rehan@ponder.io>
Date: Tue, 22 Aug 2023 14:29:53 -0700
Subject: [PATCH 2/2] Remove dependencies from setup.py

---
 setup.py | 18 ++++++------------
 1 file changed, 6 insertions(+), 12 deletions(-)

diff --git a/setup.py b/setup.py
index 596e2f5f..100d8925 100644
--- a/setup.py
+++ b/setup.py
@@ -4,13 +4,13 @@ import versioneer
 with open("README.md", "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
-dask_deps = ["dask>=2.22.0", "distributed>=2.22.0"]
+dask_deps = []
 # ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
 # pydantic<2: https://github.com/modin-project/modin/issues/6336
-ray_deps = ["ray[default]>=1.13.0,!=2.5.0", "pyarrow>=7.0.0", "pydantic<2"]
-unidist_deps = ["unidist[mpi]>=0.2.1,<=0.4.1"]
-spreadsheet_deps = ["modin-spreadsheet>=0.1.0"]
-sql_deps = ["dfsql>=0.4.2", "pyparsing<=2.4.7"]
+ray_deps = []
+unidist_deps = []
+spreadsheet_deps = []
+sql_deps = []
 all_deps = dask_deps + ray_deps + unidist_deps + spreadsheet_deps
 
 # Distribute 'modin-autoimport-pandas.pth' along with binary and source distributions.
@@ -47,13 +47,7 @@ setup(
     url="https://github.com/modin-project/modin",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    install_requires=[
-        "pandas>=2,<2.1",
-        "packaging>=21.0",
-        "numpy>=1.20.3",
-        "fsspec>=2021.07.0",
-        "psutil>=5.8.0",
-    ],
+    install_requires=[],
     extras_require={
         # can be installed by pip install modin[dask]
         "dask": dask_deps,
-- 
2.32.0 (Apple Git-132)
