From 1bb16308c4e4a5b0d1b3f8b788635b5e7c58c183 Mon Sep 17 00:00:00 2001
From: Vasilij Litvinov <vasilij.n.litvinov@intel.com>
Date: Thu, 27 Jan 2022 13:56:26 +0300
Subject: [PATCH 2/2] Remove dependencies from setup.py to be handled by conda

Signed-off-by: Vasilij Litvinov <vasilij.n.litvinov@intel.com>
---
 setup.py | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/setup.py b/setup.py
index a1935df7..350c9d02 100644
--- a/setup.py
+++ b/setup.py
@@ -4,11 +4,12 @@ import versioneer
 with open("README.md", "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
-dask_deps = ["dask>=2.22.0,<2022.2.0", "distributed>=2.22.0,<2022.2.0"]
-ray_deps = ["ray[default]>=1.4.0", "pyarrow>=4.0.1"]
-remote_deps = ["rpyc==4.1.5", "cloudpickle", "boto3"]
-spreadsheet_deps = ["modin-spreadsheet>=0.1.0"]
-sql_deps = ["dfsql>=0.4.2", "pyparsing<=2.4.7"]
+# all dependencies are to be handled by meta.yaml
+dask_deps = []
+ray_deps = []
+remote_deps = []
+spreadsheet_deps = []
+sql_deps = []
 all_deps = dask_deps + ray_deps + remote_deps + spreadsheet_deps
 
 setup(
@@ -22,7 +23,7 @@ setup(
     url="https://github.com/modin-project/modin",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    install_requires=["pandas==1.4.1", "packaging", "numpy>=1.18.5", "fsspec"],
+    install_requires=[],
     extras_require={
         # can be installed by pip install modin[dask]
         "dask": dask_deps,
@@ -32,5 +33,4 @@ setup(
         "sql": sql_deps,
         "all": all_deps,
     },
-    python_requires=">=3.8",
 )
-- 
2.30.2.windows.1

